src/Momento.Sdk.Incubating/Requests/CollectionTtl.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+namespace Momento.Sdk.Incubating.Requests
+{
+    /// <summary>
+    /// Represents the desired behavior for managing the TTL on collection
+    /// objects (dictionaries, lists, sets) in your cache.
+    ///
+    /// For cache operations that modify a collection, there are a few things
+    /// to consider.  The first time the collection is created, we need to
+    /// set a TTL on it.  For subsequent operations that modify the collection
+    /// we may choose to update the TTL in order to prolong the life of the
+    /// cached collection object, or we may choose to leave the TTL unmodified
```

Comment:
```suggestion
    /// you may choose to update the TTL in order to prolong the life of the
    /// cached collection object, or you may choose to leave the TTL unmodified
```
When we say on the first write "we need to set a TTL on it" I think that demonstrates it is a service constraint. These latter 2 events' dispositions are chosen by the user though so I'm thinking "you"s on these?